crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -190,6 +191,8 @@ pub struct CumulocityConverter {
 
     supported_operations: SupportedOperations,
     pub operation_handler: OperationHandler,
+
+    units: Units,
```

Comment:
Wondering if we should keep a `meta` wrapper at this level which internally contains the unit definitions, instead of keeping `units` directly here, since units is just one aspect of the metadata.